ts/store/reducers/backendStatus/remoteConfig.ts

```diff
@@ -409,3 +410,13 @@ export const isIOMarkdownDisabledForMessagesAndServices = (
     O.chainNullableK(ioMarkdown => ioMarkdown.disabledForMessagesAndServices),
     O.getOrElse(() => false)
   );
+
+export const pnMessageServiceIdSelector = createSelector(
```

Comment:
This selector returns a string or undefined so we do not need the createSelector to avoid re-renderings